entry_types/scrolled/package/src/frontend/Backdrop.js

```diff
@@ -146,13 +157,20 @@ function BackgroundVideo(props) {
   }, [playerActions])
 
   return (
-    <VideoPlayer isPrepared={isPrepared}
-                 playerState={playerState}
-                 playerActions={playerActions}
-                 id={props.video}
-                 textTracksDisabled={true}
-                 fit="cover"
-                 loop={true}
-                 playsInline={true} />
+    <>
+      <VideoPlayer isPrepared={isPrepared}
+                   playerState={playerState}
+                   playerActions={playerActions}
+                   id={video.permaId}
+                   textTracksDisabled={true}
+                   fit="cover"
+                   loop={true}
+                   playsInline={true} />
+      <MotifArea key={video.permaId}
```

Comment:
**question:** Is it realistic to have some spec that tests some effect/s of this addition?

Reply:
Good idea. I added tests that ensure that the `onMotifAreaUpdate` callback is invoked. The specifics of when/how the callback is called are already covered in the `MotifArea` tests. Here, the fact that the callback is called merely acts as proof that a motif area has been rendered and that this now also happens for video files.